modular_skyrat/modules/marines/code/mod.dm

```diff
@@ -0,0 +1,91 @@
+// 1 2 3 4 marine corps marine corps
+
+/datum/mod_theme/marines
+	name = "marine"
+	desc = "Developed by Nanotrasen in collaboration with multiple high-profile contractors, this specialized suit is made for high-intensity combat."
+	extended_desc = "A black and blue suit of Nanotrasen design made to be utilized by corporate space marines \
+		in active combat situations where standard gear won't cut it. Lightweight composite armor plating over \
+		a strong exoskeleton ensures that no speed is sacrificed for protection, and a variety of unique \
+		modules keep the wearer efficient during active combat situations. When response teams fail, \
+		you're the backup's backup - the desperate measures."
+	default_skin = "marine"
+	armor = list(MELEE = 50, BULLET = 50, LASER = 50, ENERGY = 50, BOMB = 50, BIO = 100, FIRE = 100, ACID = 50, WOUND = 20) //honestly quite incredible
+	atom_flags = PREVENT_CONTENTS_EXPLOSION_1
+	resistance_flags = FIRE_PROOF|ACID_PROOF
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	siemens_coefficient = 0
+	slowdown_inactive = 0.5
+	slowdown_active = 0
+	complexity_max = DEFAULT_MAX_COMPLEXITY + 10 //drr drr drr
+	allowed_suit_storage = list(
+		/obj/item/flashlight,
+		/obj/item/tank/internals,
+		/obj/item/ammo_box,
+		/obj/item/ammo_casing,
+		/obj/item/restraints/handcuffs,
+		/obj/item/assembly/flash,
+		/obj/item/melee/baton,
+		/obj/item/gun/ballistic,
+		/obj/item/melee/hammer,
+	)
+	skins = list(
+		"marine" = list(
+			MOD_ICON_OVERRIDE = 'modular_skyrat/modules/marines/icons/mod.dmi',
+			MOD_WORN_ICON_OVERRIDE = 'modular_skyrat/modules/marines/icons/wornmod.dmi',
+			HELMET_FLAGS = list(
+				UNSEALED_LAYER = NECK_LAYER,
+				UNSEALED_CLOTHING = SNUG_FIT,
+				SEALED_CLOTHING = THICKMATERIAL|STOPSPRESSUREDAMAGE|BLOCK_GAS_SMOKE_EFFECT|FLASH_PROTECTION_WELDER,
+				UNSEALED_INVISIBILITY = HIDEFACIALHAIR,
+				SEALED_INVISIBILITY = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDESNOUT,
+				SEALED_COVER = HEADCOVERSMOUTH|HEADCOVERSEYES|PEPPERPROOF,
+			),
+			CHESTPLATE_FLAGS = list(
+				UNSEALED_CLOTHING = THICKMATERIAL,
+				SEALED_CLOTHING = STOPSPRESSUREDAMAGE,
+				SEALED_INVISIBILITY = HIDEJUMPSUIT|HIDETAIL,
+			),
+			GAUNTLETS_FLAGS = list(
+				UNSEALED_CLOTHING = THICKMATERIAL,
+				SEALED_CLOTHING = STOPSPRESSUREDAMAGE,
+				CAN_OVERSLOT = TRUE,
+			),
+			BOOTS_FLAGS = list(
+				UNSEALED_CLOTHING = THICKMATERIAL,
+				SEALED_CLOTHING = STOPSPRESSUREDAMAGE,
+				CAN_OVERSLOT = TRUE,
+			),
+		),
+	)
+
+/obj/item/mod/control/pre_equipped/marine
+	theme = /datum/mod_theme/marines
+	applied_cell = /obj/item/stock_parts/cell/bluespace
+	initial_modules = list(
+		/obj/item/mod/module/storage/large_capacity,
+		/obj/item/mod/module/emp_shield/advanced,
```

Comment:
```suggestion
		/obj/item/mod/module/emp_shield,
```
adv feels like a bit much
(do for the others as well)

Reply:
That’s fair, they have bluespace cells